docs/developer-docs/setup/best-practices/architecture.mdx

```diff
@@ -0,0 +1,96 @@
+# Architecture 
+
+## Overview 
+
+One of the most important decisions when designing a dapp is the determining the project's architecture. Three commonly used project architectures include:
+
+- **Single canister projects**: Contain a single canisters that contains the entire project's functionality. This may be a single service, a frontend webpage, or both within a single canister. Benefits of single canister projects include:
```

Comment:
```suggestion
- **Single canister projects**: Consists of a single canister that contains the entire project's functionality. This may be a single service, a frontend webpage, or both within a single canister. Benefits of single canister projects include:
```